.github/workflows/flaky-test-monitor-integration.yml

```diff
@@ -1,71 +0,0 @@
-name: Flaky Test Monitor Integration Test
```

Comment:
removed "unit test" workflows for skipped and flaky tests (they were duplicating the real workflows but just not uploading the data)